NEWS

```diff
@@ -11,9 +11,140 @@
 ### Features:
 ### Bugfixes:
 
+## 1.15.0-rc2 (July 27, 2023)
+### Features:
+#### RDMA CORE (IB, ROCE, etc.)
+* Implemented is_reachable_v2 for IB interfaces
+#### Build
+* Enabled build with binutils 2.40
+* Added versioned dependency to switch between packages with the same names
+
+### Bugfixes:
+#### UCP
+* Reverted the change in lanes selection to fix wire compatibility broken in 1.15.0-rc1
```

Comment:
https://github.com/openucx/ucx/pull/9244#discussion_r1275874176
@yosefe the compatibility was broken in rc1, and fixed in rc2. Thus, it shouldn't be mentioned in final news for 1.15.0.
Is the line is ok now?

Reply:
Better remove this line IMO

Reply:
Removed